Service/src/main/java/Users/Admin.java

```diff
@@ -0,0 +1,19 @@
+package Users;
+
+import TIcket.Ticket;
+import org.example.SetClassId;
+
+public class Admin extends User {
+    private static int classId;
+    private static String role = "ADMIN";
```

Comment:
Why this field is static?
Also looks like "ADMIN" "USER" could be enum type

Reply:
Role "ADMIN" is common for all object of class ADMIN